.github/workflows/docker.yml

```
@@ -123,14 +123,17 @@ jobs:
       - name: Install Docker Buildx
         uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
 
-      - name: Login to the registry
-        run: >
-          echo ${{ secrets.GHCR_TOKEN }} |
-          docker login ${{ env.CONTAINER_NAMESPACE }}
-          --username abergs
-          --password-stdin
+      - name: Setup DCT
```

To make things clearer:

```suggestion
      - name: Setup Docker Content Trust
```